pkg/actuators/machineset/controller.go

```diff
@@ -121,7 +121,8 @@ func reconcile(machineSet *machinev1.MachineSet) (ctrl.Result, error) {
 
 	// TODO: get annotations keys from machine API
 	machineSet.Annotations[cpuKey] = strconv.FormatInt(instanceType.VCPU, 10)
-	machineSet.Annotations[memoryKey] = strconv.FormatInt(instanceType.MemoryMb, 10)
+	// See https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory
+	machineSet.Annotations[memoryKey] = fmt.Sprintf("%sMi", strconv.FormatInt(instanceType.MemoryMb, 10))
```

Comment:
These are functionally equivalent
```suggestion
	machineSet.Annotations[memoryKey] = fmt.Sprintf("%dMi", nstanceType.MemoryMb)
```

Reply:
perfect, i'll update it